### PR TITLE
GLS edit of Tech Exercise ii (CI/CD) completed.

### PR DESCRIPTION
**Pull request by nmuller66**

### CODE REVIEW

File: docs/slides/content/tech-exercise-ii.md [hunk 1]
```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
...
-* Q&A
+* Q & A
 * Real World Stories
 * Pull Requests Welcome 🦄
 
 
 
 #### WHO - Pipelines <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->
+<!--Tech Edit - this slide is empty except for the heading. -->
```
Comment (ckavili) [inline docs/slides/content/tech-exercise-ii.md:257]: @springdo do you have any screenshots you can upload? I also can dig a bit deeper and bring out something :D

File: docs/slides/content/tech-exercise-ii.md [hunk 2]
```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
...
 
 
 
 #### WHO - App of Apps <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->
+<!--Tech Edit - this slide is empty except for the heading. -->
```
Comment (ckavili) [inline docs/slides/content/tech-exercise-ii.md:263]: @springdosame goes for this one :D

File: docs/slides/content/tech-exercise-ii.md [hunk 3]
```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
...
-* **Continuous Deployment/Release**: One step further now that we are sure that this is ready to deploy to production then we can rely on features toggle, dark launches or other deployment strategy to promote to other critical environments asap using these techniques for the new features/fixes won't risk the operation.
-
+* **Continuous Integration**: Integrate several changes into a shared repository and rely heavily on testing to make sure that the changes and commits work and have passed all tests (unit testing / code quality / code conduct).
+* **Continuous Delivery**: The goal is to have a working artifact that's ready to be deployed at any time. It relies on testing and automation to make sure that it has passed all functional tests and can be manually deployed (push button).
+* **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, is to rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```
Comment (ckavili) [inline docs/slides/content/tech-exercise-ii.md:270]:
```suggestion
* **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, we can now rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

File: docs/slides/content/tech-exercise-ii.md [hunk 4]
```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
...
+<!--TechEdit: This last bullet didn't make sense as written, it was sort of a long run-on. I've rewritten, please check. -->
```
Comment (ckavili) [inline docs/slides/content/tech-exercise-ii.md:271]:
```suggestion
```

**Comment by ckavili** [comment]
let's merge this one after the screenshots are provided :)

File: docs/slides/content/tech-exercise-ii.md (hunk 3 above)
Comment (nmuller66) [inline docs/slides/content/tech-exercise-ii.md:270]:
```suggestion
* **Continuous Deployment/Release**: After we are sure that this artifact is ready to deploy to production, we can take a further step, relying on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

Comment (nmuller66) [reply docs/slides/content/tech-exercise-ii.md:270]: The grammar is not quite right in the suggestion. If we leave out the clause set off by commas, we have "A further step, ..., now we can..." which isn't correct.

Comment (nmuller66) [inline docs/slides/content/tech-exercise-ii.md:270]:
```suggestion
* **Continuous Deployment/Release**: Now that we are sure this artifact is ready to deploy to production, we can take a step further and rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

**Comment by ckavili** [comment]
thanks for the updates @nmuller66 ! we'll find the screenshots for the missing parts and then close the issue 🙏

**Comment by ckavili** [comment]
we got some screenshots, merging this  👍

**Pull request merged**